x-pack/plugins/reporting/public/components/report_listing.tsx

```diff
@@ -41,17 +41,17 @@ export interface Job {
   type: string;
   object_type: string;
   object_title: string;
-  created_by?: string;
+  created_by?: string | false;
```

Comment:
This one still kinda bugs me, but understand _why_ we have it

Reply:
yeah, it would have been nice to map this differently from the beginning